Git-Collabration-Example/ViewController.swift

```diff
@@ -9,11 +9,30 @@ import UIKit
 
 class ViewController: UIViewController {
 
+    @IBOutlet weak var clickDisplay: UIButton!
+    @IBOutlet weak var commentService: UILabel!
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
+        
+        
     }
-
+    
+    
+    @IBAction func showPage(_ sender: UIButton, forEvent event: UIEvent) {
+        let currentLabel = commentService.text
+        if (currentLabel == "You just click a button. ") {
+            commentService.text = "You click the button again"
+        }
+        else
+        {
+            commentService.text = "You just click a button"
+        }
+        
+        commentService.sizeToFit()
+    }
```

Comment:
These codes conflict with the master code so they might be merged or deleted.

Reply:
I'm working on the conflicts and will be resolved in the next update

Reply:
Now Done